docs/content/gds/members.en.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Accessing Members"
+date: 2022-01-03T14:02:37-05:00
+lastmod: 2022-01-03T14:02:37-05:00
+description: "Accessing Other TRISA Members"
+weight: 70
+---
+
+The `TRISAMembers` service is an experimental service that provides extra access to the directory service for verified TRISA members. Only members of the TRISA network (e.g. members who have been issued valid identity certificates) can access other members of the network using this service. *Note: Once validated, this service will be moved into the official TRISA specification.*
+
+## The `TRISAMembers` Service
+
+This section describes the protocol buffers for the `TRISAMembers` endpoint, which can be found [here](https://github.com/trisacrypto/directory/blob/main/proto/gds/members/v1alpha1/members.proto). This file can be compiled into the language of your choice ([example for Go](https://github.com/trisacrypto/directory/tree/main/pkg/gds/members/v1alpha1)). *Note: You will need to download and install the protocol buffer compiler if you have not already.*
+
+The `TRISAMembers` service expects as input a `ListRequest` and returns a `ListReply`.
+
+```proto
+service TRISAMembers {
+    // List all verified VASP members in the Directory Service.
+    rpc List(ListRequest) returns (ListReply) {};
+}
+```
+
+### `ListRequest`
```

Comment:
````suggestion
## The `TRISAMembers` Service

This section describes the protocol buffers for the `TRISAMembers` endpoint, which can be found [here](https://github.com/trisacrypto/directory/blob/main/proto/gds/members/v1alpha1/members.proto). This file can be compiled into the language of your choice ([example for Go](https://github.com/trisacrypto/directory/tree/main/pkg/gds/members/v1alpha1)). *Note: You will need to download and install the protocol buffer compiler if you have not already.*

Currently, the `TRISAMembers` service is comprised of only a single RPC - the `List` RPC. Other experimental, secure RPCs may be made available in the future.

```proto
service TRISAMembers {
    // List all verified VASP members in the Directory Service.
    rpc List(ListRequest) returns (ListReply) {};
}
```

## Listing Verified Members

The `List` RPC returns a paginated list of all _verified_ TRISA members to facilitate TRISA peer lookups. The RPC expects as input a `ListRequest` and returns a `ListReply`.

### `ListRequest`
````